CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Parser reset between DCS escapes
 - Parser stopping at unknown DEC private modes/SGR character attributes
 - Block selection appending duplicate newlines when last column is selected
+- Font size querying not matching renderer font size
```

Comment:
This documents what caused the bug, not what bug we're trying to fix. We should state the effect this has on the user and what they can expect to work properly now.